src/engraving/tests/compat114_tests.cpp

```diff
@@ -70,7 +70,7 @@ TEST_F(Engraving_Compat114Tests, articulations) {
 }
 
 TEST_F(Engraving_Compat114Tests, chord_symbol) {
-    compat("chord_symbol");
+    //compat("chord_symbol");
```

Comment:
Why is this test disabled? Wouldn't it be better to fix it?

Reply:
Disabled locally (they fail locally for me)
Turned it on.. thanks!